examples/hockey/hockey_game.py

```diff
@@ -20,10 +17,11 @@ def __init__(self, players: list[IPlayer]) -> None:
         """
 
         self.env = h_env.HockeyEnv()
+        self.player_1_id = players[0].id
+        self.player_2_id = players[1].id
 
         # number of rounds played per game and current score
         self.remaining_rounds: int = 4
-        self.score = [0, 0]
         # Bool weather players play in default orientation or sides are swapped.
         # Alternates between rounds.
         self.sides_swapped = False
```

Comment:
I think it would be easier if you would just swap `self.player_1_id` and `self.player_2_id`. This is possible now, because everything is given in dictionary.

Reply:
Yeah, that's better. I changed both